TelCo.ColorCoder/ColorReferenceManual.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    /// class defined to hold the two colors of clor pair
+    class ColorReferenceManual
+    {
+        public static void printColorManual()
+		{
+			for(int num1 = 0; num1 < 5; num1++)
```

Comment:
Avoid hard-coding the limits like `5` in the loop

Reply:
Sure ,I'll update it.